README.md

```diff
@@ -37,40 +78,29 @@ To use this provider binary, you have a few different options:
 
 To learn more about using a local build of a provider, you can look at the [documentation on writing custom providers](https://www.terraform.io/docs/extend/writing-custom-providers.html#invoking-the-provider) and the [documentation on how Terraform plugin discovery works](https://www.terraform.io/docs/extend/how-terraform-works.html#discovery)
 
-## Using the provider
 
-For production use, you should constrain the acceptable provider versions via configuration,
-to ensure that new versions with breaking changes will not be automatically installed by 
-`terraform init` in future. As this provider is still at version zero, you should constrain 
-the acceptable provider versions on the minor version.
+## Usage
 
-If you are using Terraform CLI version 0.12.x, you can constrain this provider to 0.15.x versions 
-by adding a `required_providers` block inside a `terraform` block.
-```
-terraform {
-  required_providers {
-    tfe = "~> 0.15.0"
-  }
-}
-```
+[Create a user or team API token in Terraform Cloud/Enterprise](https://www.terraform.io/docs/cloud/users-teams-organizations/api-tokens.html), and use the token in the provider configuration block:
 
-If you are using Terraform CLI version 0.11.x, you can constrain this provider to 0.15.x versions 
-by adding the version constraint to the tfe provider block.
-```
+```hcl
```

Comment:
TIL that Github supports syntax highlighting for HCL!

Reply:
Totally guessed, I honestly have no idea lol

Reply:
Looks like it! https://github.com/github/linguist/blob/65d078bab8a8bafc5c53614be5c3fec42c2897f0/lib/linguist/languages.yml#L2016-L2030